sql/parser/sql.y

```diff
@@ -3596,10 +3588,7 @@ reserved_keyword:
 | CONSTRAINT
 | CREATE
 | CURRENT_CATALOG
-| CURRENT_DATE
```

Comment:
In general, you shouldn't be touching the `reserved_keyword` rule. Why was this change necessary? I'd like to see it reverted.